.circleci/config.yml

```diff
@@ -24,7 +24,7 @@ jobs:
 
   e2eTestAWS: &e2eTestAWS
     docker:
-      - image: quay.io/giantswarm/docker-terraform-and-stuff:latest
+      - image: quay.io/giantswarm/docker-terraform-and-stuff:gotemplate
```

Comment:
You probably want to switch back to latest before the merge.

Reply:
sure